### ISSUE TITLE
Format Code not modifying the first line selected (unless it is selected all the way to the left)

### ISSUE THREAD

**Issue by jeffschwMSFT** [OmniSharp/omnisharp-vscode#214]
![format](https://cloud.githubusercontent.com/assets/9561583/14621128/88171e80-0575-11e6-8d33-08fa7ccfcfd8.gif)

Here is the sample code (though it is trivial):
using System;
using System.Collections.Generic;

namespace ConsoleApplication
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Dictionary<string, string> context = new Dictionary<string, string>() {
                {"a", "some string"}};

```
                    int foo = 1;
                     bool b = false;

                        Console.WriteLine("{0} {1} {2}", context["a"], foo, b);
    }
}
```

}

If you select anywhere between the "n" in "int" to "e" in "Console" and then request VS Code to format the code, it will do the last 2 lines, but not the first.

C# Extension 1.0.3-rc2
Mac OS X

**Comment by DustinCampbell** [comment]
Good bug -- thanks!

**Comment by rchande** [comment]
@akshita31 Can you take a look?

**Comment by akshita31** [comment]
![ezgif-1-b627375cd4](https://user-images.githubusercontent.com/12554141/33450490-ccc61ea6-d5c0-11e7-8bef-08cb32533992.gif)

This issue does not reproduce now.

**Comment by akshita31** [comment]
The issue has been resolved with the change : https://github.com/OmniSharp/omnisharp-roslyn/pull/1043 .

**Comment by akshita31** [comment]
This issue has been fixed in the 1.14 milestone. We just released a beta of 1.14 - https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4. Follow [these instructions](https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4) to a beta release of C# for VS Code.
